scripts/windows/addAllToPool.bat

```diff
@@ -0,0 +1,3 @@
+set "SQL_QUERY= insert into interview_interviewpool select id from members_user"
+
+docker exec -it swecc-server-db-1 psql -U root postgres -c "%SQL_QUERY%"
```

Comment:
can do this without sql so it also works in prod (not that we should ever run this in prod)

```bash
container_id=$1
cmd="put_all_users_in_pool"
docker exec -it $container_id python server/manage.py $cmd
```

or w/e it is.